test/tools/ClientCertificateCheck/dockerfile

```diff
@@ -0,0 +1,18 @@
+# Stage 1: Build the Application
```

Comment:
@iSazonov I will remove it and .dockerignore as well as the docker info in the readme.